spec/helpers.lua

```diff
@@ -983,6 +986,7 @@ local function grpc_client(host, port, opts)
       end
 
       local opts = gen_grpcurl_opts(pl_tablex.merge(t.opts, args.opts, true))
+      ngx.log(ngx.ERR, string.format(t.cmd_template, opts, service))
```

Comment:
Is this a forgotten debug message?

Reply:
Thanks for the reminder, I will remove it.